rl78/add.c

```diff
@@ -0,0 +1,8 @@
+int main(int argc, char *argv[])
```

Comment:
add a comment with the command used to build it.

Reply:
Don't merge yet, we need more complex test

Reply:
I asked to compile `float_ex1.c`